spidermon/contrib/scrapy/pipelines.py

```diff
@@ -3,19 +3,30 @@
 import json
 from io import BytesIO
 from collections import defaultdict
+import logging
 
 from scrapy.exceptions import DropItem, NotConfigured
 from scrapy.utils.misc import load_object
 from scrapy.exporters import JsonLinesItemExporter
 from scrapy import Field, Item
 from scrapy.utils.python import to_native_str
 
-from spidermon.contrib.validation import SchematicsValidator, JSONSchemaValidator
-from spidermon.contrib.validation.jsonschema.tools import get_schema_from
+from spidermon.contrib.validation import SchematicsValidator, JSONSchemaValidator, CerberusValidator
+from spidermon.contrib.validation.utils import get_schema_from
 from schematics.models import Model
 
 from .stats import ValidationStatsManager
 
+logx = logging.getLogger(__name__)
```

Comment:
Probably, these shouldn't be here

Reply:
True that. Will remove it. 
Will add tests as well.

Reply:
Thanks @ejulio